statsd/statsd.go

```diff
@@ -55,6 +55,12 @@ traffic instead of UDP.
 */
 const UnixAddressPrefix = "unix://"
 
+/*
+WindowsPipeAddressPrefix holds the prefix to use to enable Windows Named Pipes
+traffic instead of UDP.
+*/
+const WindowsPipeAddressPrefix = `\\.\pipe\`
```

Comment:
Does this need to be public ?

Reply:
It absolutely does not, but then again neither do the UDS ones. I was just following what's there.

Reply:
We have too many public method/attribute in this package. The next major is going to make everything that doesn't need to be public private.

I guess we could keep it public for consistency for now since it's a const.